localization/english/culture/wc_cultural_maa_innovations_l_english.yml

```
@@ -18,3 +19,187 @@
+ 
+ innovation_silithid:0 "Silithid"
+ innovation_silithid_desc:0 "Silithids are tough and sophisticated insectoid creatures. The chitin plates of their bone armor are intertwined in such a way that it makes it incredibly hard. Lumped in huge numbers, the silithids are devouring swarms. Stopping at nothing but the demise of other life on Azeroth."
+
+ innovation_snow_hunter:0 "Snow Hunters"
+ innovation_snow_hunter_desc:0 "Our kind had to adapt to the elements in order to survive. The best of our warriors are the hunters who keep us safe."
+    
+ innovation_thornguard:0 "Thornguards"
+ innovation_thornguard_desc:0 "The Thornguards are the finest warriors from our tribe, who defend us from any outsiders. They are heavily armed and armored. They will stop any who dare to trespass or any who dare to not submit."
+
+ innovation_tidehunter:0 "Tidehunters"
+ innovation_tidehunter_desc:0 "Mglrmglmglmgl. Mmmm Mmmurlok. Uuua n Mrgllll glrrm gl!"
+ 
+ innovation_transcendent:0 "Transcendent Avatars"
+ innovation_transcendent_desc:0 "Pandaren revere transcendents as embodiments of wisdom: the spirit of the universe made flesh, residing in the shell of a pandaren host. These are the pandaren’s ultimate heroes: Spiritual avatars capable of summoning vast storms of lighting, smashing mountains with their terrible fists, and breathing winds into existence."
+    
+ innovation_ursa:0 "Ursas"
+ innovation_ursa_desc:0 "Ursas are the biggest, strongest and most feral of furbolgs. They tower above humans and orcs, and can snap elven necks with single paw swipes. They are immensely strong and tough, and focus on improving their natural, bearlike natures."
+
+ innovation_war_dinosaur:0 "War Dinsosaurs"
+ innovation_war_dinosaur_desc:0 "Once a dominant species throughout the planet, they have become an endangered remnant of an ancient world that prove to be a force to be reckoned with. These great beasts are as dangerous as they are majestic. Our tamers have successfully learned how to communicate and tame these grand creatures."
+
+ innovation_wolfrider:0 "Wolfriders"
+ innovation_wolfrider_desc:0 "These fierce warriors use the speed and ferocity of their mounts to make them a deadly enemy. Faster than those who attack on foot, they have perfected the tactics of tearing through columns of enemy soldiers like a hot wind."
+ 
+ ######################
+ # Early Medieval Era #
+ ######################
+  
+ innovation_bearpaws:0 "Bearpaws"
+ innovation_bearpaws_desc:0 "Though they are only lightly armed, the bearpaws serve a big punch if provoked as strong and skilled fighters that they are."
+
+ innovation_blademaster:0 "Blademasters"
+ innovation_blademaster_desc:0 "The battle-hardened orcish Blademasters are few in number, yet they form an elite force within the Horde. Skilled swordsmen, they once belonged to the doomed clan of the Burning Blade, which, thanks to demonic corruption, is on the brink of its own destruction at all times."
```

Last bit was changed a little, as it felt a little odd.
```suggestion
 innovation_blademaster_desc:0 "The battle-hardened orcish Blademasters are few in number, yet they form an elite force within the Horde. Skilled swordsmen, they once belonged to the doomed clan of the Burning Blade, which, thanks to demonic corruption, teeters on the brink of destruction."
```